code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -103,20 +103,9 @@ emp_act
 		return 1
 	if(wear_suit)
 		if(wear_suit.IsShield() && (prob(50)))

```

Comment:
I think @Firecage tried to do that for shields but it got rejected for some reason, I'll try and dig up his PR to see why